search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/filter/DecisionDefinitionFilterTransformer.java

```diff
@@ -51,7 +58,8 @@ public SearchQuery toSearchQuery(final DecisionDefinitionFilter filter) {
 
   @Override
   public List<String> toIndices(final DecisionDefinitionFilter filter) {
-    return List.of("operate-decision-8.3.0_alias");
+    final String indexPrefix = DecisionIndex.getIndexNameWithPrefix(prefix);
```

Comment:
Hi @nathansandi, Houssain just pointed me to this PR. I've actually addressed the same issue from a slightly different angle in https://github.com/camunda/camunda/pull/25048/commits/dfc35dc89f298f5effc108baf495317430ac3e5d. The main goal there is to return the IndexDescriptor from toIndices() directly, so that I have access to tenancy information on them. But it also means that I get the full index name here for free.
What do you think of this approach?

Reply:
I notices that we sometimes return the alias and sometimes the index itself, but I believe this can be unified: we already create an alias for all indices and templates and so we can always query the alias.

I want to return the IndexDescriptor from the toIndices so I have access to configuration on them. This also gets rid of the hardcoded indices.

Reply:
It's still work in progress, but I would look like this: https://github.com/camunda/camunda/pull/25048/files#diff-7da04f0e010698982a4f835e1438845c9879f0acd372487c963a1135203b85a0

Reply:
Sounds good to me ;) 
I am going tag @ChrisKujawa for visibility, so as you are covering this on your PR.
Please, let me know if you need something from my side

I gonna leave this PR open for while, just in case any decision changes.

Reply:
@nathansandi sorry I had not time to look into this, do you still need my input?

Reply:
@nathansandi Do you have time to review my PR? If the full set of changes is too much, I'm particular interested in feedback on the first commit, which passes index descriptors into filter transformers.

Reply:
Hello @emulbreh  :) I have checked this morning. So far looks good to me.